demos/dws-equity-style-box-time-series/demo.js

```diff
@@ -0,0 +1,73 @@
+import { getPostmanFile } from '../utils/postman-localstorage.js';
+
+getPostmanFile(displayEquitySectorsBreakdown, 'postmanEnvironmentDWS');
+
+const loadingLabel = document.getElementById('loading-label');
+
+async function displayEquitySectorsBreakdown (postmanJSON) {
+
+    const connector = new HighchartsConnectors.MorningstarDWS.InvestmentsConnector({
+        postman: {
+            environmentJSON: postmanJSON['postmanEnvironmentDWS']
+        },
+        security: {
+            id: '0P00006W6Q'
+        },
+        converters: {
+            EquityStyleBox: {
+                startDate: '2025-01-01',
+                endDate: '2025-12-01'
+            }
+        }
+    });
+
+    await connector.load();
+
+    const table = connector.getTable('TimeSeries');
+
+    Highcharts.stockChart('container', {
```

Comment:
Could you replace this chart with a simple Grid? This looks good, but we should probably spend more time on it (choose the right visual, series type etc etc). See for example `Highcharts Dashboards DataGrid + Morningstar XRay Connector (Benchmark)`